examples/bulk_water_langevin_ncmc.py

```diff
@@ -0,0 +1,315 @@
+# gradually turn off a single water, displace, then turn on
```

Comment:
note: although this script used NVT rather than NPT, I don't think it was impacted by the bug fixed in https://github.com/proteneer/timemachine/pull/1168 (since it didn't use ixngroup)